modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/ExecutionServiceImpl.java

```diff
@@ -165,6 +178,7 @@ public static <RowT> ExecutionServiceImpl<RowT> create(
     public ExecutionServiceImpl(
             ClusterNode localNode,
             MessageService msgSrvc,
+            TopologyService topSrvc,
```

Comment:
in this case, we could don't pass localNode parameter

Reply:
Removed localNode parameter from signature